assets/js/job-editor/JobEditorMain.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
```

Comment:
Can we find better names for `JobEditorMain` and `JobEditor`? It's quite untidy

I presume we have the distinction becuse `JobEditorMain` is aware of the phoenix infrastructure, but `JobEditor` is not? So the `Main` file is just like a mount or wrapper for the app

The architecture is great but the name is off :)

Reply:
Yup I agree. any suggestions? 
`JobEditor` & `JobEditorComponent` ?